test/puppetlabs/puppetdb/scf/migrate_test.clj

```diff
@@ -57,11 +57,11 @@
 
         (testing "should attempt a partial migration if there are migrations missing"
           (clear-db-for-testing!)
-          ;; We are using migration 13 here because it is isolated enough to be able
+          ;; We are using migration 28 here because it is isolated enough to be able
```

Comment:
this change makes sense to me because the modified test is about out-of-order migrations, and migration 13 uses the report column in latest_reports, which this migration drops.

The new migration should probably have a test though IMO.